wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -908,9 +916,21 @@ public final class SessionManager: NSObject, SessionManagerType {
         }
     }
 
-    private func configureAnalytics(for userSession: ZMUserSession) {
-        analyticsService.switchUser(userSession.analyticsUser)
-        userSession.analyticsEventTracker = analyticsService
+    func configureAnalytics(for userSession: ZMUserSession) {
```

Comment:
what about making it async instead and wait until it's complete to call `delegate?.sessionManagerDidCompleteInitialSync(for: activeUserSession)` instead

Reply:
done 783e4f85f8be8e6168fee4427bba1da400286d9c